### ISSUE TITLE
Background color of buttons in new form creation is WAY to close to background

### ISSUE THREAD

**Issue by Shelob9** [CalderaWP/Caldera-Forms#725]
I'm having trouble seeing it on some screens and I'm only minor color blind. It's also going to fail any accessibility test for color contrast. I'm making them white background, green text.

**Comment by Shelob9** [comment]
Close via 83b72c7
